From 2f22333b5d39651cf0b2b973396faca510317d6c Mon Sep 17 00:00:00 2001
From: Levente Polyak <levente@leventepolyak.net>
Date: Sat, 27 Jan 2024 23:01:59 +0100
Subject: [PATCH] fix: handle AUTH_TLS reply to FEAT for all TLS variants

Send 'AUTH SSL' in reply to the FEAT command when any of the TLS
versions is enabled.
---
 features.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/features.c b/features.c
index 1212980..3a60b88 100644
--- a/features.c
+++ b/features.c
@@ -22,7 +22,7 @@ handle_feat(struct vsf_session* p_sess)
     {
       vsf_cmdio_write_raw(p_sess, " AUTH SSL\r\n");
     }
-    if (tunable_tlsv1)
+    if (tunable_tlsv1 || tunable_tlsv1_1 || tunable_tlsv1_2 || tunable_tlsv1_3)
     {
       vsf_cmdio_write_raw(p_sess, " AUTH TLS\r\n");
     }
-- 
2.43.0

